<?xml version="1.0"?>
<robot name="simrad_es200-7c">

  <link name="ek80_nav">
    <visual>
    </visual>
  </link>

  <link name="ek80">
    <visual>
      <geometry>
        <cylinder length="0.0752" radius="0.06"/>
      </geometry>
      <origin rpy="0 1.57075 0" xyz="-0.0376 0 0"/>
    </visual>
  </link>

  <joint name="nav_to_transducer" type="fixed">
    <parent link="ek80_nav"/>
    <child link="ek80"/>
    <origin rpy="0 1.57075 0"/>
  </joint>
</robot>